.idea/codeStyles/Project.xml

```diff
@@ -8,9 +8,6 @@
     <option name="LINE_SEPARATOR" value="&#10;" />
     <option name="WRAP_WHEN_TYPING_REACHES_RIGHT_MARGIN" value="true" />
     <option name="FORMATTER_TAGS_ENABLED" value="true" />
-    <AndroidXmlCodeStyleSettings>
-      <option name="USE_CUSTOM_SETTINGS" value="true" />
-    </AndroidXmlCodeStyleSettings>
```

Comment:
Is it okay if add these lines again?

Reply:
Yes, you can restore the original file. As a rule of thumb I am suggesting to never submit any files you did not intentionally change. Always have a look at the "Files changed" tab before submitting a PR.

Reply:
This branch of my repository was completely botched. Please check this [PR](https://github.com/Catrobat/Catroid/pull/3555). Thank You.
Cheers,
Darshit.